v1.2.yml

```diff
@@ -1608,3 +1784,259 @@ definitions:
       email:
         type: string
         x-nullable: true
+
+  StudentObject:
```

Comment:
My knowledge of swagger is limited, but is there a way to avoid having to duplicate definitions across files? If so, is this something that we would want to do?